### PR TITLE
Bump to jenkins-2.89.4

### PR DESCRIPTION
**Pull request by ramereth**
Also do the following security updates on the plugins:

- credentials-binding
- junit
- scm-api
- workflow-support

### CODE REVIEW

File: attributes/plugins.rb [hunk 1]
```diff
@@ -69,7 +69,7 @@
   pipeline-utility-steps:1.4.0
   plain-credentials:1.4
   run-condition:1.0
-  scm-api:2.2.0
+  scm-api:2.2.6
```
Comment (doublej472) [inline attributes/plugins.rb]: I don't see a security update for this plugin, but there isn't any major, breaking changes, so this is fine.

Comment (doublej472) [reply attributes/plugins.rb]: I'm looking at https://plugins.jenkins.io/scm-api for the changes BTW

Comment (ramereth) [reply attributes/plugins.rb]: I believe this was a dependency that was pulled in for one of the plugins that got updated.

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.9.16.  Have a nice day!

**Pull request merged**